### ISSUE TITLE
Simplify workflow for inviting users by email

### ISSUE THREAD

**Issue by dantefromhell** [humhub/humhub#5651]
### Is your feature request related to a problem? Please describe.
The current workflow for inviting users by email is confusing, as one has to first select *Add new user* to find the *Send Invite* button.

Once the "Send Invite" button is visible there is also a larger form to fill out the user details, including a field for email. But this field is not related to the "Send Invite" field.

### Describe the solution you'd like
Move the "Send Invite" button to the user overview (humhub.tld/admin/user/list). 
If positioned next to the "Add new User" button the two different workflows can be started as needed from the same page.

A possible further clarification could be renaming the "Send Invite" button to "Send eMail Invite" (if space permits).

**Comment by Meeques** [comment]
Hi @dantefromhell, is the invitation via email working at your instance? A colleague who I invited via email just told me that they didn't receive an email.

**Comment by dantefromhell** [comment]
> is the invitation via email working at your instance? 

Yes inviting via email works from the user administration view as well as from the Space view for me. Tested on v1.11
